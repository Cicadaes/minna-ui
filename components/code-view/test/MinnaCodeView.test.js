'use strict';

const MinnaCodeView = require('../src/MinnaCodeView.html');

describe('MinnaCodeView component', () => {
  it('renders correctly with defaults', () => {
    const target = document.createElement('div');
    new MinnaCodeView({ target });
    expect(target.innerHTML).toMatchSnapshot();
  });
});
