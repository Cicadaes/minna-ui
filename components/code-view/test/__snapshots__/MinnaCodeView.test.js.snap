// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MinnaCodeView component renders correctly 1`] = `""`;
